assemblies/common-resources/src/main/resources-filtered/etc/org.apache.karaf.features.xml

```diff
@@ -60,11 +60,11 @@
 
         <!-- [PPP-4108] Use of vulnerable component jackson-mapper-asl-1.9.2.jar CVE-2017-7525, CVE-2017-15095, CVE-2017-15095 -->
         <bundle
-            originalUri="mvn:org.codehaus.jackson/jackson-core-asl/[1.5.0,${codehaus-jackson.version})"
-            replacement="mvn:org.codehaus.jackson/jackson-core-asl/${codehaus-jackson.version}" mode="maven" />
+            originalUri="mvn:org.codehaus.jackson/jackson-core-asl/[1.5.0,1.9.14)"
```

Comment:
We should be replacing every version until "${fasterxml-jackson}", not "1.9.14"!
So it should be like this:
`          originalUri="mvn:org.codehaus.jackson/jackson-core-asl/[1.5.0,${fasterxml-jackson})"`

If one leaves the code as is, a jar whose version falls in the interval between 1.9.14 and 2.14.1 (current value for ${fasterxml-jackson}) would not be replaced, which could bring some inconsistencies.